docs/core/tools/dotnet-tool-install.md

```diff
@@ -0,0 +1,96 @@
+---
+title: dotnet tool install command - .NET Core CLI
+description: The dotnet tool install command installs the specified .NET Core Global Tool on your machine.
+author: mairaw
+ms.author: mairaw
+ms.date: 04/25/2018
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.workload: 
+  - dotnetcore
+---
+# dotnet tool install
+
+[!INCLUDE [topic-appliesto-net-core-2-1plus.md](../../../includes/topic-appliesto-net-core-2-1plus.md)]
+
+## Name
+
+`dotnet tool install` - Installs the specified [.NET Core Global Tool](global-tools.md) on your machine.
+
+## Synopsis
+
+```
+dotnet tool install <PACKAGE_ID> <-g|--global> [--configfile] [-f|--framework] [--source-feed] [-v|--verbosity] [--version]
+dotnet tool install <PACKAGE_ID> <--tool-path> [--configfile] [-f|--framework] [--source-feed] [-v|--verbosity] [--version]
+dotnet tool install [-h|--help]
+```
+
+## Description
+
+The `dotnet tool install` command provides a way for you to install .NET Core Global Tools on your machine. To use the command, you either have to specify that you want a user-wide installation using the `--global` option or you specify a path to install it using the `--tool-path` option.
+
+Global Tools are installed in the following directories by default when you specify the `-g` (or `--global`) option:
+
+| OS          | Path                          |
+|-------------|-------------------------------|
+| Linux/macOS | `$HOME/.dotnet/tools`         |
+| Windows     | `%USERPROFILE%\.dotnet\tools` |
+
+## Arguments
+
+`PACKAGE_ID`
+
+ID of the NuGet package that contains the .NET Core Global tool to install.
+
+## Options
+
+`--configfile <FILE>`
+
+The NuGet configuration (*nuget.config*) file to use.
+
+`-f|--framework <FRAMEWORK>`
+
+Specifies the [target framework](../../standard/frameworks.md) to install the tool for.
```

Comment:
when does someone need to specify this @KathleenDollard?

Reply:
no, not until a breaking runtime came out.

Producers can construct packages targeting multiple TFMs, say, a single package with a tool for netcoreapp2.1 and netcoreappVNEXT. Or even produce a package with a TFM different from the one used in the CLI.

The CLI utilize NuGet to pick the fittest TFM. The CLI has the notion of a bundled TFM, which indicates the TFM that the CLI itself targets. CLI will always pass that TFM to NuGet and NuGet will write asset file with filtered TFM. CLI does not prob the disk, instead it finds asset by reading the asset file.

This approach does not work when the tool is packaged with netcoreapp and say net46, because the CLI will always pick the netcoreapp TFM. In order to work around this, the consumer will have to provide the TFM to install the command. Note that this should not be a problem if the package contains only the net46 TFM.

Also, when the user has the 2.1 CLI installed and then installs the 2.1 runtime, the 2.1 version of the tool will continue to be selected, as that’s the CLI bundled version. But the user can obtain the netcoreapp3.0 version of tool by specifying TFM in the command line. Again using the escape hatch we defined above. We use the TFM bundled version because it has the highest likelihood of succeeding, since that tool will target the runtime that the CLI itself is using to run.

Reply:
@wli3 @KathleenDollard 

Ok, I understand the concept that inside the package, you can have multiple TFMs supported and by specifying this, you can effectively select one. But it would be good to add to the command description, some concise explanation of why you might need to specify this. the netcoreapp and other TFMs, for example, seems like a good scenario. Or maybe an explanation that CLI will try to choose the best TFM for you, so you rarely need this?

Reply:
Whatcha' think about this:

While most Global Tools are not framework specific, framework specific tools are supported. If a framework is specified, it is used to select the correct version of the tool. If a framework is not specified, the framework of the currently running .NET SDK is used. 

The framework is specified via its TFM, which you can find [here](../../standard/frameworks.md).

In most cases, you do not need the `--framework` switch.

Reply:
"While most Global Tools are not framework specific, framework specific tools are supported"

This is kind of wrong. All tools are TFM specific. And we only support netcoreapp2.1. Maybe

"SDK will try to choose most appropriate TFM by default . However, this can be overwritten by this flag""